client/views/edit/edit.js

```diff
@@ -0,0 +1,168 @@
+  },
+  newLengthFifth() {
+    return (Template.instance().data.table.new_length === 86400) ? 'selected' : '';
+  },
+  newLengthSixth() {
+    return (Template.instance().data.table.new_length === 604800) ? 'selected' : '';
+  },
+  // stale_length
+  staleFirst() {
+    return (Template.instance().data.table.stale_length === 900) ? 'selected' : '';
+  },
+  staleSecond() {
+    return (Template.instance().data.table.stale_length === 1800) ? 'selected' : '';
+  },
+  staleThird() {
+    return (Template.instance().data.table.stale_length === 3600) ? 'selected' : '';
+  },
+  staleFourth() {
+    return (Template.instance().data.table.stale_length === 86400) ? 'selected' : '';
+  },
+  staleFifth() {
+    return (Template.instance().data.table.stale_length === 604800) ? 'selected' : '';
+  },
+  staleSixth() {
+    return (Template.instance().data.table.stale_length === 2592000) ? 'selected' : '';
+  },
+  staleLengthSeventh() {
+    return (Template.instance().data.table.stale_length === 31557600) ? 'selected' : '';
+  },
+  // hidden
+  visibilityFirst() {
+    return (Template.instance().data.table.hidden === false) ? 'selected' : '';
+  },
+  visibilitySecond() {
+    return (Template.instance().data.table.hidden === true) ? 'selected' : '';
+  },
+  // social
+  socialFirst() {
+    return (Template.instance().data.table.social === true) ? 'selected' : '';
+  },
+  socialSecond() {
+    return (Template.instance().data.table.social === false) ? 'selected' : '';
+  },
+});
+
+Template.edit.events({
+  'click .editsubmitbutton': function (event, template) {
+    const table = Template.instance().data.table;
+    const thresholdSelect = document.getElementsByName('threshold')[0];
+    const threshold = thresholdSelect[thresholdSelect.selectedIndex].value;
+    const questionSelect = document.getElementsByName('max_question')[0];
+    const maxQuestion = questionSelect[questionSelect.selectedIndex].value;
+    const responseSelect = document.getElementsByName('max_response')[0];
+    const maxResponse = responseSelect[responseSelect.selectedIndex].value;
+    const lengthSelect = document.getElementsByName('new_length')[0];
+    const redLength = lengthSelect[lengthSelect.selectedIndex].value;
+    const staleSelect = document.getElementsByName('stale_length')[0];
+    const stale = staleSelect[staleSelect.selectedIndex].value;
+    const hiddenSelector = document.getElementsByName('visibility')[0];
+    const isHidden = (hiddenSelector[hiddenSelector.selectedIndex].value === 'hidden');
+    const socialSelector = document.getElementsByName('social')[0];
+    const social = (socialSelector[socialSelector.selectedIndex].value === 'on');
+    // console.log("New parameters: ",threshold, maxQuestion, maxResponse, redLength, stale, isHidden, social);
+    Meteor.call('editadv', table._id, threshold, maxQuestion, maxResponse, redLength, stale, isHidden, social, (error, result) => {
```

Comment:
These instance attributes can be collected into a single object, which will clean up the code and allow for a bit more flexibility in the future.